code-challenges.test.js

```diff
@@ -38,11 +91,34 @@ const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
+const onlyOddLtoG = (arr1Q2) => {
+    return arr1Q2.filter(value => {
+        if(value % 2 !== 0 && typeof(value) === "number"){
+            return arr1Q2
+        }
+    }).sort((a, b) => a - b)
+}
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
+//I'll write out a jest test ensuring the following 4 methods are in place
+//describe,,expect,toEqual
+// describe will utilize the name 
+//describe  :arraySum
+//it        :will take in an array of numbers and return a new array that added all priori indexes
+//expect    :numbersToAdd1 = [2, 4, 45, 9]
+//toEqual   :[2, 6, 51, 60]
+
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("arraySum", () => {
+    it("will take in an array of numbers and return a new array that added all prior indexes", () => {
+        expect(arraySum(numbersToAdd1)).toEqual([2, 6, 51, 60])
+        // expect(arraySum(numbersToAdd2)).toEqual([0, 7, -1, 11])
```

Comment:
You should consider passing all the expect statements for all the variables in the question.